inc/commonitilobject.class.php

```diff
@@ -3942,7 +3942,7 @@ static function getActorIcon($user_group, $type) {
 
       switch ($user_group) {
          case 'user' :
-            $icontitle = __s('User').' - '.$type; // should never be used
+            $icontitle = addslashes(User::getTypeName(1)).' - '.$type; // should never be used
```

Comment:
why ?

Reply:
The alternative would be:
```suggestion
            $icontitle = _sn('User', 'Users', 1).' - '.$type; // should never be used
```

But I think it is better to rely on `getTypeName()` rather than copying the original string.

Reply:
ok